src/test/java/org/jabref/logic/integrity/EditionCheckerTest.java

```diff
@@ -3,15 +3,35 @@
 import java.util.Optional;
 
 import org.jabref.model.database.BibDatabaseContext;
-
+import org.jabref.model.database.BibDatabaseMode;
+import org.jabref.model.entry.BibEntry;
+import org.junit.jupiter.api.BeforeEach;
 import org.junit.jupiter.api.Test;
 
 import static org.junit.jupiter.api.Assertions.assertEquals;
 import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.assertNotEquals;
 import static org.junit.jupiter.api.Assertions.assertTrue;
 
 public class EditionCheckerTest {
 
+    private EditionChecker checker;
+    private EditionChecker checkerb;
+    private BibEntry entry;
+    private BibDatabaseContext bibtex;
+    private BibDatabaseContext biblatex;
+
+    @BeforeEach
+    void setUp() {
```

Comment:
Here, the `@BeforeEach` is OK, because there is more happening than just creating objects.